src/expertsystem/reaction/conservation_rules.py

```diff
@@ -615,6 +615,15 @@ def __spin_couplings(spin1: _Spin, spin2: _Spin) -> Set[_Spin]:
     )
 
 
+def arange(
```

Comment:
Just a comment: I put that into `quantum_numbers.py` before, since it felt a bit lost in combinatorics or conservation_rules.

Reply:
Ok 1921f54